frontend/src/component/pages/shared/PreferenceToggle.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { Component, ReactChild } from 'react';
+import { Toggle } from '../../ui/Toggle';
+
+interface IProps {
+  toggleLabel: ReactChild;
+  defaultIsEnabled: boolean;
+  onToggleClick?: (enabled: boolean) => void;
+}
+
+const DEFAULT_PROPS = {
+  defaultIsEnabled: false
+};
+
+export class PreferenceToggle extends Component<IProps> {
+  static defaultProps: Partial<IProps> = DEFAULT_PROPS;
+
+  render() {
+    return (
+      <div className={'preference-toggle'}>
+        <Toggle
+          defaultIsEnabled={this.props.defaultIsEnabled}
+          onClick={this.props.onToggleClick}
+        ></Toggle>
+        <span className={'toggle-label'}>{this.props.toggleLabel}</span>
```

Comment:
```suggestion
        <div className={'toggle-label'}>{this.props.toggleLabel}</div>
```

Reply:
Changing to div requires us to use flex layout on `preference-toggle`

Reply:
Already made `preference-toggle` a `flex`, but I have now moved it into its own `scss` file which I will push momentarily.